### PR TITLE
TLC-253 Create Footer PO

### PR DESCRIPTION
**Pull request by cyxoi1**

### CODE REVIEW

**Comment by cyxoi1** [comment]
second attempt

File: pageObjects/footer.page.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import BasePage from './../pageObjects/Base.page';
+
+class FooterPage extends BasePage {
+  get footerText() {
+    return $('[class="mt-auto mb-15rem"]').getText();
+  }
```
Comment (jstepiko) [inline pageObjects/footer.page.js:6]: Should be an element, but not a text from it.
getText() should be used in the test, not here, but in spec. => I'd name it just `footer`

File: pageObjects/footer.page.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  get termsOfServicePageHeader() {
+    return $('h1');
+  }
+
+  get privacyPolicyPageHeader() {
+    return $('h1');
+  }
+
```
Comment (jstepiko) [inline pageObjects/footer.page.js:45]: They are methods => they should be methods here or tests in spec

Comment (cyxoi1) [reply pageObjects/footer.page.js]: All done

**Pull request merged**